src/main/resources/static/js/MainWindow.js

```diff
@@ -1,670 +1,756 @@
+/*global optimizeforsource, mappingData, currentDelimiter, currentHeaderlines, preview, addRow, delLastRow,
+ * resetMapping, showstep1, isExcel, getLocations, initDatastream, loadMapping, returnRows*/
 /**
+ * 
  * shows the popup to create a new Thing
  */
 function showThingModal() {
-    var $modal = $("#indexfooter").find("button:eq(0)");
-    $modal.attr("onclick", "createThing()");
-    $modal.html("Create Thing");
-    $modal.show();
-    modal("dialog", "thing.html", getLocations, "Create a Thing");
+	var $modal = $("#indexfooter").find("button:eq(0)");
+	$modal.attr("onclick", "createThing()");
+	$modal.html("Create Thing");
+	$modal.show();
+	modal("dialog", "thing.html", getLocations, "Create a Thing");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['getLocations' is not defined.](https://app.codacy.com/gh/FraunhoferIOSB/ChillImport/pullRequest?prid=4781409)